roles/satellite/tasks/define.yml

```diff
@@ -30,17 +30,17 @@
 - include_tasks:
     file: internals/contentview-promote.yml
   vars:
-    target_lce_name: "{{ g_test_lifecycleenvironment }}"
+    target_lce_name: "{{ test_lifecycleenvironment }}"
     source_lce_name: Library
-  when: g_test_lifecycleenvironment_defined
+  when: test_lifecycleenvironment_defined
 
 # promote contentview from test->prod
 - include_tasks:
     file: internals/contentview-promote.yml
   vars:
-    target_lce_name: "{{ g_prod_lifecycleenvironment }}"
-    source_lce_name: "{{ g_test_lifecycleenvironment }}"
-  when: g_test_lifecycleenvironment_defined and g_prod_lifecycleenvironment_defined
+    target_lce_name: "{{ prod_lifecycleenvironment }}"
+    source_lce_name: "{{ test_lifecycleenvironment }}"
+  when: test_lifecycleenvironment_defined and prod_lifecycleenvironment_defined
```

Comment:
I think you should directly use the "variable is defined" statement instead of using the facts.
This indirection makes things a bit harder to understand and the amount of text it adds here is not excessive.

This would mean removing completely these facts and the file that creates these facts.

Reply:
OK. I updated the PR.